tests/compiler.js

```diff
@@ -1395,6 +1395,10 @@
                 expect(res).to.be('&><\'"');
             });
 
+            render('{{ foo }}', { foo: null }, { autoescape: true }, function (err, res) {
+                expect(res).to.be('');
+            });
```

Comment:
add `render('{{ foo | safe }}', { foo: '<html>' }, { autoescape: true }` test for safe filter.